lib/theme.js

```diff
@@ -23,7 +23,7 @@ export default {
     formGrey: '#C2C2C2',
     badgeGrey: '#9FAFB9',
     mutedOrange: '#F9A606',
-    onSurface: '#D5D9E0',
+    onSurface: '#708390',
```

Comment:
Is the existing color wrong? 🤔

Reply:
I think so, i don't see it being used anywhere, The one defined on "dai-ui" is 708390, we were overwritting it for some reason

Reply:
nevermind, changed